spk/mitmproxy/Makefile

```diff
@@ -0,0 +1,29 @@
+SPK_NAME = mitmproxy
+SPK_VERS = 0.18.2
+SPK_REV = 1
+SPK_ICON = src/mitmproxy.png
+
+WHEELS = src/requirements.txt
```

Comment:
To keep the flow in the file, should we perhaps format it as BUILD_DEPENDS, DEPENDS, WHEELS, SPK_DEPENDS, essentially in the order that the entries will be processed?